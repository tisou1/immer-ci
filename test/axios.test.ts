
import { describe, it, expect, vi } from 'vitest'
import axios from 'axios'

function buyApples() {
  return axios('https://www.fastmock.site/mock/6f92f55a6b8b6a1bdf0a2f35dcbe01ec/data1/1')
}


it('axios', async () => {
  // toEqual returns a promise now, so you HAVE to await it
  const res = await buyApples()
  expect(res.data).toMatchSnapshot()
  // await expect(buyApples()).resolves.toMatchSnapshot() // jest API
  // await expect(buyApples()).resolves.to.equal({ id: 1 }) // chai API
})
