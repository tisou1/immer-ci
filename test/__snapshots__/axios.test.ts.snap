// Vitest Snapshot v1

exports[`axios 1`] = `
{
  "list": [
    "React",
    "ES6",
    "javaScript",
    "vue",
    "python",
  ],
}
`;
